worker-taskstowing/src/main/java/com/microfocus/caf/worker/taskstowing/TaskStowingWorker.java

```diff
@@ -0,0 +1,159 @@
+            LOGGER.error(TaskStowingWorkerFailure.TRACKING_INFO_NOT_PRESENT);
+            return createFailureResult(TaskStowingWorkerFailure.TRACKING_INFO_NOT_PRESENT);
+        }
+        final String jobTaskIdFromTrackingInfo = trackingInfo.getJobTaskId();
+        if (Strings.isNullOrEmpty(jobTaskIdFromTrackingInfo)) {
+            LOGGER.error(TaskStowingWorkerFailure.JOB_TASK_ID_NOT_PRESENT);
+            return createFailureResult(TaskStowingWorkerFailure.JOB_TASK_ID_NOT_PRESENT);
+        }
+        final JobTaskId jobTaskId = JobTaskId.fromMessageId(jobTaskIdFromTrackingInfo);
+        final String partitionId = jobTaskId.getPartitionId();
+        if (Strings.isNullOrEmpty(partitionId)) {
+            LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_PARSE_PARTITION_ID_FROM_JOB_TASK_ID);
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_PARSE_PARTITION_ID_FROM_JOB_TASK_ID);
+        }
+        final String jobId = jobTaskId.getJobId();
+        if (Strings.isNullOrEmpty(jobId)) {
+            LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_PARSE_JOB_ID_FROM_JOB_TASK_ID);
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_PARSE_JOB_ID_FROM_JOB_TASK_ID);
+        }
+
+        try {
+            // workerTask.getContext() should not be null, but it's possible if a worker is passed a message with a context map that
+            // has a key that is not the same as the servicePath, in which case the Worker Framework will pass a null context:
+            // https://github.com/WorkerFramework/worker-framework/blob/develop/worker-core/src/main/java/com/hpe/caf/worker/core/WorkerTaskImpl.java#L124
+            final byte[] contextBytes = workerTaskData.getContext() != null
+                ? workerTaskData.getContext()
+                : OBJECT_MAPPER.writeValueAsBytes(Collections.<String, byte[]>emptyMap());
+            final byte[] trackingInfoBytes = OBJECT_MAPPER.writeValueAsBytes(workerTaskData.getTrackingInfo());
+            final byte[] sourceInfoBytes = OBJECT_MAPPER.writeValueAsBytes(workerTaskData.getSourceInfo());
+
+            databaseClient.insertStowedTask(
+                partitionId,
+                jobId,
+                workerTaskData.getClassifier(),
+                workerTaskData.getVersion(),
+                workerTaskData.getData(),
+                workerTaskData.getStatus().name(),
+                contextBytes,
+                workerTaskData.getTo(),
+                trackingInfoBytes,
+                sourceInfoBytes,
+                workerTaskData.getCorrelationId());
+            return createSuccessResultNoOutputToQueue();
+        } catch (final JsonProcessingException jsonProcessingException) {
+            LOGGER.error(TaskStowingWorkerFailure.FAILED_TO_SERIALIZE_TASK, jsonProcessingException);
+            return createFailureResult(TaskStowingWorkerFailure.FAILED_TO_SERIALIZE_TASK, jsonProcessingException);
+        } catch (final Exception exception) {
```

Comment:
We will need more here.  We need to try to identify whether the exception was caused by a transient issue, and if it was to throw a `TaskRejectedException` for it (which is the Worker Framework equivalent of `DocumentWorkerTransientException`).  Carefully study the Job Tracking Worker's [JobTrackingWorkerReporter.java](https://github.com/JobService/job-service/blob/v3.5.0-source/worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java) to see what types of issues it treats as transient.  Note that the list is slightly different during the process of establishing a connection.  We'll have to work out how/if the Hikari and Jdbi wrappers effect how the exceptions are presented to us.